client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -40,7 +40,7 @@ registerBlockType( 'a8c/simple-payments', {
 								return;
 							}
 
-							const result === parseInt( id, 10 );
```

Comment:
:trollface:

Reply:
I just really wanted to test that fancy commit UI no matter what. ;-)